user-guide/settings/browsers.md

```diff
@@ -9,14 +9,18 @@ previous_url: /user-guide/main-menu
 
 # Browsers Menu
 
-The **Browsers** sub-menu provides an option for changing the default Fiddler's browser from Google Chrome to other third-party Chromium browsers.
+The **Select Browser** sub-menu provides an option for changing the default Fiddler's browser from Google Chrome to other third-party Chromium browsers.
 
-- The **Automatically detect path to Google Chrome** option finds and uses the Google Chrome for the [**Independent Browser Capturing**]({% slug capture-traffic-get-started %}#independent-browser-capturing) option.
+- The **Automatically detect path to Google Chrome** option finds and uses the Google Chrome for the [**independent browser capturing mode**]({% slug capture-traffic-get-started %}#independent-browser-capturing).
 
 - The **Path to Browser** option allows you to change the default browser by setting a path to a different Chromium browser. Fiddler Everywhere supports most Chromium-based browsers (like Edge, Brave, Vivaldi, etc.).
 
+The **Browser Data** sub-menu provides an option for choosing whether the [independent browser capturing mode]({%slug browsers-settings-submenu%}) should always use a clean instance or an instance that persists the input data.
 
-The following figure displays the settings of the **Browsers** menu.
+- The **Use clean instance** option instructs Fiddler to always use a new clean instance for the [**independent browser capturing mode**]({% slug capture-traffic-get-started %}#independent-browser-capturing). No data persists when reusing the capturing mode.
 
+- The **Persist data in a specific directory** option allows you to set a custom directory to persist input data for the [**independent browser capturing mode**]({% slug capture-traffic-get-started %}#independent-browser-capturing). 
```

Comment:
it is not only input data - it persists all settings, like installed extensions for example